### PR TITLE
feat: add send ckb from multisig address

### PR DESCRIPTION
**Pull request by yanguoyu**
1: for 1-of-n multisig address

https://user-images.githubusercontent.com/12881040/160287108-dc566daf-cda9-4947-b857-762ca3138015.mov

2: for m-of-n multisig address when m>1

https://user-images.githubusercontent.com/12881040/160287184-351850fe-d013-444f-8d82-4321704ccf40.mov

### CODE REVIEW

**Review by EmmaZHONG1024 — APPROVED** [review]

**Comment by Keith-CY** [comment]
Dropdown menu could be refined, there's blank background leaked between two menu items
image
And its item's gray background is some kind of weird.

The dark background of the form in a dialog could be removed too
image

**Comment by yanguoyu** [comment]
> Dropdown menu could be refined, there's blank background leaked between two menu items image And its item's gray background is some kind of weird.
> 
> The dark background of the form in a dialog could be removed too image

fixed like this
image
image

File: packages/neuron-ui/src/components/PasswordRequest/index.tsx [hunk 1]
```diff
@@ -116,6 +121,13 @@ const PasswordRequest = () => {
           throw new PasswordIncorrectException()
         }
       }
+      const handleSendMultiTxRes = ({ status }: { status: number }) => {
+        if (isSuccessResponse({ status })) {
+          window.location.reload()
```
[the remark this answers is not in the document]
Comment (yanguoyu) [reply packages/neuron-ui/src/components/PasswordRequest/index.tsx:126]: Use reload to close dialogs.

[the remark this answers is not in the document]
Comment (Keith-CY) [reply packages/neuron-ui/src/components/PasswordRequest/index.tsx:126]: Could we close dialogs and update data manually. In some cases, the window will be blank temporarily and make the app not so native.

[the remark this answers is not in the document]
Comment (yanguoyu) [reply packages/neuron-ui/src/components/PasswordRequest/index.tsx:126]: ok, i can transfer `closeDialog` function as props to close dialog.

**Comment by Keith-CY** [comment]
I accidentally post this [feedback](https://github.com/nervosnet/neuron-public-issues/issues/5) in other channel

**Comment by Keith-CY** [comment]
image
When I press `Send`, the `Total Amount` has an error by default

And the title of window is in Chinese while Language of Neuron is set to English

**Comment by Keith-CY** [comment]
When I input `123` in Amount, the Window broken

https://user-images.githubusercontent.com/7271329/161220722-de295139-aa31-4db5-97bb-5012af0cd89a.mov

**Comment by yanguoyu** [comment]
> I accidentally post this [feedback](https://github.com/nervosnet/neuron-public-issues/issues/5) in other channel

Maybe I made a mistake in resolving the conflict.I have resolved it right now

**Comment by yanguoyu** [comment]
> image
> 
> When I press `Send`, the `Total Amount` has an error by default
> 
> And the title of window is in Chinese while Language of Neuron is set to English

I think i should disabled `send` when the balance is 0.It can resolve this

**Comment by Keith-CY** [comment]
> > image
> > When I press `Send`, the `Total Amount` has an error by default
> > And the title of window is in Chinese while Language of Neuron is set to English
> 
> I think i should disabled `send` when the balance is 0.It can resolve this

I think the real problem here is that `0 CKB` triggered an error. Ideally it should has no error while the `send` button is disabled.

**Comment by yanguoyu** [comment]
https://github.com/nervosnetwork/neuron/pull/2313#issuecomment-1085550611

**Comment by Keith-CY** [comment]
> [#2313 (comment)](https://github.com/nervosnetwork/neuron/pull/2313#issuecomment-1085550611) [#2313 (comment)](https://github.com/nervosnetwork/neuron/pull/2313#issuecomment-1085554501) fixed

With this fix, the tx fee was calculated even the tx is invalid, which should be 0
image

**Review by Keith-CY — APPROVED** [review]

**Comment by yanguoyu** [comment]
> > [#2313 (comment)](https://github.com/nervosnetwork/neuron/pull/2313#issuecomment-1085550611) [#2313 (comment)](https://github.com/nervosnetwork/neuron/pull/2313#issuecomment-1085554501) fixed
> 
> With this fix, the tx fee was calculated even the tx is invalid, which should be 0 image

fixed with https://github.com/nervosnetwork/neuron/pull/2313/commits/360a75abccacc02b0e440c0292b2c058e05affd8

**Pull request merged**